ios/kata05.xcodeproj/project.pbxproj

```diff
@@ -12,6 +12,22 @@
 		13B07FBF1A68108700A75B9A /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 13B07FB51A68108700A75B9A /* Images.xcassets */; };
 		13B07FC11A68108700A75B9A /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
 		81AB9BB82411601600AC10FF /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 81AB9BB72411601600AC10FF /* LaunchScreen.storyboard */; };
+		636F91E599DD435689DB60B8 /* AntDesign.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 8714F0FC127B406E93315F1E /* AntDesign.ttf */; };
+		E5D07E27A4C042D4A3E1BF42 /* Entypo.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C02301842EEC4CBF8F299E27 /* Entypo.ttf */; };
+		DAE171A8D2BB4AB5AC32244C /* EvilIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 27F26C522B8B493C836A0FC9 /* EvilIcons.ttf */; };
+		F71DCB55B33241279ABB97C1 /* Feather.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1D946A442B1D44A4A19BBBFE /* Feather.ttf */; };
+		21E5DDD8B10641E8BDFC954D /* FontAwesome.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 2015765BF5E546848F2220EC /* FontAwesome.ttf */; };
+		067963AEE8D3420C84CE189F /* FontAwesome5_Brands.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 9C0644D06F994B95834275C8 /* FontAwesome5_Brands.ttf */; };
+		3BD9A96BEDCE456AA276BF92 /* FontAwesome5_Regular.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 38B206D0ACE54C6D88541FAC /* FontAwesome5_Regular.ttf */; };
+		AAEDBE229FD94E719A1BF1AD /* FontAwesome5_Solid.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 00839EF74E6A4650854EA3FA /* FontAwesome5_Solid.ttf */; };
+		66E5F665CF774CCC9BCBE10D /* Fontisto.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1CC8CC5CE38443CC99419D88 /* Fontisto.ttf */; };
+		7D128670442240F3AA807B3E /* Foundation.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 153B32382F2E4463A38984CD /* Foundation.ttf */; };
+		942A027894584394847C95B4 /* Ionicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = E2BB729D95DB40CE8DB98C04 /* Ionicons.ttf */; };
+		141EB20F87764FFDB9321C87 /* MaterialCommunityIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = D8E8D2B0A9D24A7AA67B7BA2 /* MaterialCommunityIcons.ttf */; };
+		112765ACA5AF44B9A1E14175 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C94AB4ACBF324DCCA4955E87 /* MaterialIcons.ttf */; };
+		228A2758F3D849D4928ADC29 /* Octicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C4FF0E88E3AC4E32B9D87AA7 /* Octicons.ttf */; };
+		8DCCA2A7F2A64F0CBA4BAF32 /* SimpleLineIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C542EAADECAB4F5F98AC3612 /* SimpleLineIcons.ttf */; };
+		C8FDF27E203A48E5837F185C /* Zocial.ttf in Resources */ = {isa = PBXBuildFile; fileRef = CA42A6B2B63F44B5B483936A /* Zocial.ttf */; };
```

Comment:
Why are you adding multiple fonts?

Reply:
We didn't notice that. 
We installed Vector Icons and it should be added automatically.